INSTALL.rst

```diff
@@ -31,7 +31,11 @@ Mandatory
 
 gcc
 """
-- 4.9 to 5.X (depends on your current `CUDA version <https://gist.github.com/ax3l/9489132>`_)
+- 4.9 to 7 (if you want to build of Nvidia GPUs, supported compilers depend on your current `CUDA version <https://gist.github.com/ax3l/9489132>`_)
+
+  - CUDA 8.0: Use gcc 4.9
+  - CUDA 9.0-9.1: Use gcc 4.9-5
```

Comment:
I know, this is nit-picking, but I just read your change and thought CUDA 9 needs `gcc 4.9-5` as in "the fifth bug fix release of `gcc 4.9`, which surprised me. :laughing: Probably because I am used to such version numbers from Debian... :roll_eyes:  Could you plese add some spaces to make it more clear? :wink:

Reply:
you are right, that makes a lot of sense :+1: updated.